xml/System.Windows.Annotations/ContentLocatorPart.xml

```diff
@@ -211,7 +211,7 @@
         <ReturnType>System.ComponentModel.PropertyChangedEventHandler</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>This event supports the Windows Presentation Foundation (WPF) infrastructure and is not intended to be used directly from your code.</summary>
```

Comment:
Should the `<forInternalUseOnly />` tag be added?

Reply:
I explained my logic on part 1/10. But if I add that tag here, it will duplicate this info. So when we didn't actually have a summary, I left this sentence instead.